build.gradle

```diff
@@ -4,15 +4,17 @@ buildscript {
             'androidXTestCore':'1.0.0',
             'androidXTestRules':'1.1.0',
             'assertJ': '3.16.1',
-            'compose': '1.0.0-alpha11',
-            'composeNavigation': '1.0.0-alpha06',
+            'compose': '1.0.0-beta01',
+            'composeActivity': '1.3.0-alpha03',
+            'composeConstraintLayout': '1.0.0-alpha03',
+            'composeNavigation': '1.0.0-alpha08',
             'detekt': '1.7.4',
             'espresso': '3.2.0',
             'gradle': '4.2.0-alpha15',
             'junit' : '4.12',
             'junitImplementation' : '1.1.1',
-            'kotlin': '1.4.21-2',
-            'kotlinCompilerVersion': '1.4.21-2',
+            'kotlin': '1.4.30',
+            'kotlinCompilerVersion': '1.4.30',
```

Comment:
1.4.31 is out now

Reply:
That's the one I'm using on AS. However, it seems like that version doesn't work with Compose just yet - https://kotlinlang.slack.com/archives/CJLTWPH7S/p1614538297223300?thread_ts=1614518656.164200&cid=CJLTWPH7S